lib/tessel/deploy.js

```diff
@@ -54,6 +54,26 @@ Tessel.prototype.deployScript = function(opts) {
   });
 };
 
+Tessel.prototype.restartScript = function(opts) {
+  var self = this;
+  var isPush = opts.type === 'flash';
+  var filepath = isPush ? PUSH_PATH : RUN_PATH;
+
+  return new Promise(function(resolve, reject) {
+    actions.findProject(opts.entryPoint).then(function(project) {
```

Comment:
I just tested what happens when the remote file doesn't exist and it appears to fail silently:
```
➜  t2-demo  t2 restart index.js --type=flash
INFO Connecting to Tessel...
INFO Connected to Tessel-02A337692559 over LAN
INFO Running index.js...
```

I think we should explicitly check that code exists in that remote directory prior to deployment and throw an error if it doesn't.